docs/cli.rst

```diff
@@ -127,6 +127,8 @@ Each resource's ``list`` action supports ``-i/--id``, ``-l/--limit`` and
 Set Queries
 ~~~~~~~~~~~
 
+.. _set_query_ref:
```

Comment:
Plz move this above the heading for "Set Queries" and rename it to just `_set_queries`.